src/Bridges/Objective/functionize.jl

```diff
@@ -0,0 +1,43 @@
+"""
+    FunctionizeBridge{T}
+
+The `FunctionizeBridge` converts a `SingleVariable` objective into a
+`ScalarAffineFunction{T}` objective.
+"""
+struct FunctionizeBridge{T} <: AbstractBridge end
+function bridge_objective(::Type{FunctionizeBridge{T}}, model::MOI.ModelLike,
+                           func::MOI.SingleVariable) where T
+    F = MOI.ScalarAffineFunction{T}
+    MOI.set(model, MOI.ObjectiveFunction{F}(), convert(F, func))
+    return FunctionizeBridge{T}()
+end
+
+function supports_objective_function(
+    ::Type{<:FunctionizeBridge}, ::Type{MOI.SingleVariable})
+    return true
+end
+MOIB.added_constrained_variable_types(::Type{<:FunctionizeBridge}) = Tuple{DataType}[]
+function MOIB.added_constraint_types(::Type{<:FunctionizeBridge})
+    return Tuple{DataType, DataType}[]
+end
+function MOIB.set_objective_function_type(::Type{FunctionizeBridge{T}}) where T
+    return MOI.ScalarAffineFunction{T}
+end
+
+function MOI.delete(model::MOI.ModelLike, bridge::FunctionizeBridge) end
+
+function MOI.set(::MOI.ModelLike, ::MOI.ObjectiveSense,
+                 ::FunctionizeBridge, ::MOI.OptimizationSense)
+end
```

Comment:
When is this function called?

Reply:
When the sense of the model is changed, i.e. in `MOI.get(::AbstractBridgeOptimizer, ::ObjectiveSense, ::OptimizationSense)`.
The docstring in https://github.com/JuliaOpt/MathOptInterface.jl/pull/867/ should make that clear

Reply:
It would be useful to add a comment inside the function
```Julia
# FunctionizeBridge is sense agnostic, therefore, we don't need to tell it if the sense changes.
```
Alternatively, do these need to overload `MOI.set`? Why not just call it `_update_objectivesense_of_bridge`?

Reply:
The reuse of `MOI.set` and `MOI.get` for bridges where bridges are used in place of the model of indices is common in bridges so I find it consistent, even if the meaning is not directly obvious without reading the docstring. I have added a comment to make it clearer.